### PR TITLE
[ONNX FE] ONNX Clip no default min/max consts

### PR DESCRIPTION
**Pull request by mitruska**
### Details:
 - [ONNX Clip](https://github.com/onnx/onnx/blob/master/docs/Operators.md#clip) fix default min/max Constants in ONNX frontend.
 There is no need to create defaults when max or min inputs are not provided.
 Skipping the defaults allow to avoid overflow on different types max/min limits.

(Previously proposed here: https://github.com/openvinotoolkit/openvino/pull/3608)
### Tickets:
 - 69816

### CODE REVIEW

File: ngraph/frontend/onnx/frontend/src/op/clip.cpp [hunk 1]
```diff
@@ -30,30 +30,17 @@ OutputVector clip(const Node& node) {
 namespace set_11 {
 OutputVector clip(const Node& node) {
```
Comment (tomdol) [inline ngraph/frontend/onnx/frontend/src/op/clip.cpp:31]: According to the ONNX spec https://github.com/onnx/onnx/blob/master/docs/Operators.md#inputs-1---3 `min` is an input at index 1 and `max` at index 2. You're retrieving them in a different order here. Are you sure this still works like previously? Also, I think you should add a test that covers a case when `min` is optional in the model and `max` is explicitly passed in.

Comment (mitruska) [reply ngraph/frontend/onnx/frontend/src/op/clip.cpp:31]: The order is correct and aligned with the common clamp/clip implementation (https://docs.openvino.ai/latest/openvino_docs_ops_activation_Clamp_1.html).

The math formula is:
image

There are several onnx clip test backend tests covering default inputs (all passed, [CI log](https://dev.azure.com/openvinoci/dldt/_build/results?buildId=195634&view=logs&j=8e6fb299-7deb-52f1-2f4a-df91230aba00&t=f812e137-0344-51cb-e8b0-0ab752070263&l=1443)), but I can add more.

**Review by t-jankowski — CHANGES_REQUESTED** [review]

File: ngraph/test/onnx/onnx_import.in.cpp [hunk 2]
```diff
@@ -3674,6 +3674,18 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_clip_inbounds) {
     test_case.run();
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, onnx_clip_min_const_default_max) {
```
[the remark this answers is not in the document]
Comment (mitruska) [reply ngraph/test/onnx/onnx_import.in.cpp]: Done

**Review by t-jankowski — APPROVED** [review]

**Review by tomdol — APPROVED** [review]

**Comment by mitruska** [comment]
Closed as it could be no backward and onnx compatible change (like clamping infinitive).
Alternative solution in PR https://github.com/openvinotoolkit/openvino/pull/8489.

**Pull request closed without merging**